Report.md

```diff
@@ -72,10 +66,17 @@ Stakeholders refer to individuals/entities that have an interest in the developm
 
 Design Thinking Approach
 ========================
-*3. Describe your approach from a Design Thinking principles perspective. Your Design Thinking approach to address the problem with artefacts*
-
 ## **Brainstormed Ideas and Reflection**
-*TODO*
+The following features were concluded from the brainstorming session. 
+Feature | Justification
+--- | ---
+Enhance Interactive Voice Response unit | Gain initial understanding of customer needs without the need for an in-person RM answering the call. This, in addition to the addition of hold music, were the best ideas ideated to increase customer retention in calls while no RMs were available. We concluded that other ideas such as playing ads while customers were on hold were abandoned as they would lead to negative customer experience.
```

Comment:
are we keeping "This, in addition..." derp sentence?

Reply:
Can we make it shorter - In addition, hold music respective of customer demographic was suggested to increase customer retention in calls while no RMs were available. We concluded that other ideas such as playing ads while customers were on hold were abandoned as they would lead to negative customer experience.

Reply:
Or we could get rid of it if it takes up space hahah can't believe we didn't catch it

Reply:
Copied it straight across ahaha good catch, probably just going to remove it - doesn't add a lot